### PR TITLE
Android Study Jam 2018 new page

### PR DESCRIPTION
**Pull request by miquelbeltran**
Please @iluu and @Anialeinchen check the contents and review :) 

Any comments are welcome, if you feel we should change something, write differently, add more info, etc. all welcome

### CODE REVIEW

File: android-study-jam.md [hunk 1]
```diff
@@ -6,118 +6,108 @@ isStaticPost: true
...
-- [Android Basics: User Interface](https://www.udacity.com/course/android-development-for-beginners--ud837) 
-- [Android Basics: User Interaction](https://www.udacity.com/course/android-basics-user-input--ud836)
+#### How we work
 
-And once registered, start the courses clicking on “Start Free Course”.
+We will meet Tuesdays from 19:00 to 21:00 for six weeks.
 
-Then, [join the next Android Study Jam on Meetup.com](https://www.meetup.com/women-techmakers-berlin/events/)
+During the six scheduled sessions, we will follow the codelab [Build your first Android app in Kotlin](https://codelabs.developers.google.com/codelabs/build-your-first-android-ap)
+and we will go step by step through all the content. See the agenda bellow.
```
Comment (laramartin) [inline android-study-jam.md]: > bellow 

should be **below**

File: android-study-jam.md [hunk 2]
```diff
@@ -6,118 +6,108 @@ isStaticPost: true
...
-The students are expected to follow the online course at home and use the study
-jam session as support. Android developers, named as coaches, will be present on
+Android developers, named as coaches, will be present on
 the study jam session to help the students.
 
-About 3 hours/week of work are required to complete the course on time.
-
-Also, the students will develop their first app during the course and will
-present the app on the graduation event.
+We recommend you to added to all sessions, but skipping a session and doing the
```
Comment (laramartin) [inline android-study-jam.md]: > added

should be **attend**?

File: android-study-jam.md [hunk 3]
```diff
@@ -6,118 +6,108 @@ isStaticPost: true
...
+- Questions and self-learning
+- End at 21:00
 
 #### Diversity and Code of Conduct
 
-The Android Study Jam is organized with collaboration of Women Techmakers
-Berlin, GDG Berlin and GDG Berlin Android. Our mission is that everyone feels
-welcome to our events no matter gender or nationality, take their first steps
+The Android Study Jam is organized by Women Techmakers Berlin.
+Our mission is that everyone feels welcome to our events no matter gender or nationality, take their first steps
```
Comment (laramartin) [inline android-study-jam.md]: I would rewrite this part only to conjugate the verbs correctly, see bold words: 

> Our mission is that everyone feels welcome to our events no matter gender or nationality, **takes** their first steps in tech and **is** part of our local community.

Comment (Anialeinchen) [reply android-study-jam.md]: How about something like this:
"Our mission is that everyone feels welcome to our events no matter gender, nationality or age.
We are a diversity supporting community and want to encourage newcomers to take their first steps in tech and become a part of the Berlin's tech scene."
"During the course you will get to know the basics of Android Development in Kotlin, the programming language that was designed to replace Java and is highly demanded on the job market. You should also consider participating even if you are not actively looking for a job. 
Learning to code by developing your first app is pretty much one of the best ways to choose."

**Pull request merged**